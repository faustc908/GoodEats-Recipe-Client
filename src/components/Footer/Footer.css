.footer {
  text-align: center;
  padding: 5px;
  border-top: 2px solid black;
  margin-top: 10px;
}

.link {
  padding-left: 5px;
}
