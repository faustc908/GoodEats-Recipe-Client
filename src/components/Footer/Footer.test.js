import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Footer from './Footer'
import ReactDOM from 'react-dom';

describe(`Footer component`, () => {

    it('renders a Footer by default', () => {
        const wrapper = shallow(<Footer />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Footer />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
})