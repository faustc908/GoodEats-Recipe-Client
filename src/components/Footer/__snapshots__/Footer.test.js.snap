// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders a Footer by default 1`] = `
<div
  className="footer"
>
  <p>
    Â©Colin Faust 2020
  </p>
  <p>
     
    Powered by
    <a
      className="link"
      href="https://developer.edamam.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Edamam
    </a>
  </p>
</div>
`;
